shadow-dom/leaktests/selection.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name='author' title='weizman' href='https://www.weizmangal.com'>
+<meta name='assert' content='Shadow DOM should not leak via Selection API'>
+<link rel='help' href='https://w3c.github.io/webcomponents/spec/shadow/'>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+</head>
+<body>
+<div id='log'></div>
+<div id='container'></div>
+</body>
+<script>
+  'use strict';
+
+  const container = document.getElementById('container');
+
+  function reset(text) {
+    const host = container.appendChild(document.createElement('div'));
+    const shadow = host.attachShadow({mode: 'closed'});
+    const child = shadow.appendChild(document.createElement('span'));
+    child.textContent = text;
+    return host;
+  }
+
+  async function dblclick(t, target) {
```

Comment:
Is it possible to drag-select? Perhaps that would work in Safari?

Reply:
yeah, drag-select is probably better in that the behavior of selection under double clicking isn't spec'ed anywhere.

Reply:
I could really use a clarification here, because precedent-wise, the code-base is overloaded with usage of `dblclick` (I should know, my implementation is 100% copy-pasted) - what makes this case specifically different so that `dblclick` not being spec'ed anywhere matters, as opposed to already existing tests?

Reply:
Furthermore, if this repo is strictly about testing spec'ed only functionality, and `dblclick` isn't spec'ed anywhere - what's this `uievents/click/dblclick_event_mouse.html` non-tentative test for?

Reply:
bde72a9279

Moved to drag-select to comply with spec to your request @rniwa (this did not however change anything in regards with Safari though @noamr , at least not locally for me)